fetch-in-react/starter-files/src/app.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars -- Remove this line. */
+import React, { useState, useEffect } from 'react';
+import UserList from './user-list';
+
+export default function App() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [users, setUsers] = useState([]);
+
+  /* your code here (hint: useEffect) */
+  const url = 'https://jsonplaceholder.typicode.com/users';
+  useEffect(() => {
+    fetch(url)
+      .then((response) => response.json())
+      .then((response) => {
+        setUsers(response);
+        setIsLoading(false);
+      })
+      .catch((error) => console.error(error));
```

Comment:
If the network request fails, please let the user known so they aren't stuck on an eternal loading screen